pkg/adapter/containers.go

```diff
@@ -252,6 +255,9 @@ func (r *LocalRuntime) RemoveContainers(ctx context.Context, cli *cliconfig.RmVa
 			Fn: func() error {
 				err := r.RemoveContainer(ctx, c, cli.Force, cli.Volumes)
 				if err != nil {
+					if errors.Cause(err) == define.ErrNoSuchCtr {
+						return nil
```

Comment:
IDK for sure, so thought I'd throw it out for consideration.  Would a logrus.debug line be helpful at some point here?

Reply:
and the previous if too at line 238

Reply:
Done